packages/liveblocks-core/src/devtools/protocol.ts

```diff
@@ -91,8 +93,16 @@ export type ClientToPanelMessage =
       storage?: readonly DevTools.LsonTreeNode[];
       me?: DevTools.UserTreeNode;
       others?: readonly DevTools.UserTreeNode[];
-    };
+    }
 
+  /**
+   * Sent whenever the ydoc is updated
+   */
+  | {
+      msg: "room::sync::ydoc";
+      roomId: string;
+      update: YDocUpdate | UpdateYDocClientMsg;
```

Comment:
Can we rename `YDocUpdate` to `YDocUpdateServerMsg`? I had to look up what it meant, and noticed it's the only one in the union without this suffix:

https://github.com/liveblocks/liveblocks/blob/a1cd3002aa42ff0e29023693b98d38c1fd0e33b7/packages/liveblocks-core/src/protocol/ServerMsg.ts#L29-L51

Reply:
fixed, note: will need to update server side at some point.